libdist/CMakeLists.txt

```diff
@@ -7,9 +7,20 @@ set(sources
         src/NetworkIOMPI.cpp
         src/NetworkLCI.cpp
 )
+
+add_library(galois_dist_async_obj OBJECT ${sources})
+target_include_directories(galois_dist_async_obj PUBLIC
```

Comment:
We should confirm that the includes for libgalois and libllvm are picked up automatically by CMake and not list them here.

Reply:
Do you mind if we defer this particular change for later?

I was planning to do this when we/I refactor our cmake usage to be strictly target based. And fix our installation :D

Reply:
That's totally fine with me. Several of these comments are meant as "we should fix this", not "we should fix this right now."